auth.php

```diff
@@ -313,6 +313,14 @@ public function pre_loginpage_hook() {
      * All the checking happens before the login page in this hook
      */
     public function loginpage_hook() {
+
+        $pluginsfunction = get_plugins_with_function('auth_saml2_loginpage_hook', 'lib.php');
```

Comment:
we claim support of 2.7 moodle for this plugin (https://moodle.org/plugins/pluginversions.php?plugin=auth_saml2).
Does get_plugins_with_function exists in Moodle 2.7 ?